lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -564,6 +606,26 @@ func (c *Client) sharedDirectoryDeleteRequest(req tdp.SharedDirectoryDeleteReque
 	return C.ErrCodeSuccess
 }
 
+//export tdp_sd_list_request
+func tdp_sd_list_request(handle C.uintptr_t, req *C.CGOSharedDirectoryListRequest) C.CGOErrCode {
+	// TODO(isaiah): WRONG
```

Comment:
What is wrong here? Is the implementation wrong and you're planning to fix it or you've just forgotten to delete this comment? :)

Reply:
Hah yes, a forgotten note to self.